frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -2122,44 +2158,82 @@ https://archiveweb.page/images/${"logo.svg"}`}
           )}
         `,
       )}
-      ${inputCol(html`
-        <btrix-time-input
-          hour=${ifDefined(this.formState.scheduleTime?.hour)}
-          minute=${ifDefined(this.formState.scheduleTime?.minute)}
-          period=${ifDefined(this.formState.scheduleTime?.period)}
-          @time-change=${(e: TimeInputChangeEvent) => {
-            this.updateFormState({
-              scheduleTime: e.detail,
-            });
-          }}
-        >
-          <span slot="label">${msg("Start Time")}</span>
-        </btrix-time-input>
-        <div class="mt-3 text-xs text-neutral-500">
-          <p class="mb-1">
-            ${msg(
-              html`Schedule:
-                <span class="text-blue-500"
-                  >${utcSchedule
-                    ? humanizeSchedule(utcSchedule)
-                    : msg("Invalid date")}</span
-                >.`,
-            )}
-          </p>
-          <p>
+      ${when(
+        this.formState.scheduleFrequency,
+        () =>
+          html`${inputCol(html`
+            <btrix-time-input
+              hour=${ifDefined(this.formState.scheduleTime?.hour)}
+              minute=${ifDefined(this.formState.scheduleTime?.minute)}
+              period=${ifDefined(this.formState.scheduleTime?.period)}
+              @time-change=${(e: TimeInputChangeEvent) => {
+                this.updateFormState({
+                  scheduleTime: e.detail,
+                });
+              }}
+            >
+              <span slot="label">${msg("Start Time")}</span>
+            </btrix-time-input>
+            ${scheduledDate(this.utcSchedule)}
+          `)}
+          ${this.renderHelpTextCol(
+            msg(`A crawl will run at this time in your current timezone.`),
+          )}`,
+        () => html`
+          ${inputCol(html`
+            <sl-input
+              name="scheduleCustom"
+              label=${msg("Cron Schedule")}
+              placeholder="@hourly"
+              value=${ifDefined(this.formState.scheduleCustom)}
+              minlength="6"
+              @sl-change=${(e: SlChangeEvent) => {
+                const input = e.target as SlInput;
+                const value = (e.target as SlInput).value;
+                const parsed = parseCron(value);
+
+                if (!value) return;
+
+                if (parsed) {
+                  input.helpText = "";
+                  input.setCustomValidity("");
+                } else {
+                  let errorMessage = "";
+
+                  try {
+                    cronstrue.toString(value);
```

Comment:
It looks like [cronstrue supports loading locales](https://github.com/bradymholt/cronstrue#individual-locales), it'd be nice to use the current locale here

Reply:
Good catch! Updated to support translations--made a note of this in a comment, it doesn't look like errors are translated so we may need to address that with custom translations.